css/style.css

```diff
@@ -8,656 +8,1472 @@
 
 body {
   font-family: "Poppins", sans-serif;
-}
-
-header {
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  align-content: center;
-  padding: 11px;
-  max-width: 375px;
-  margin: 0 auto;
-}
-
-nav {
-  display: flex;
-  align-items: center;
-  height: 50px;
-  width: 100%;
-}
-
-ul {
-  list-style: none;
-}
-
-.logo-part {
-  flex: 1;
-  padding: 20px;
-  font-weight: 800;
-  font-size: 24px;
-}
-
-.logo-text {
-  color: #6070ff;
-  list-style: none;
-}
-
-.logo-text a {
-  text-decoration: none;
-  color: #6070ff;
-}
-
-.menu-part {
-  display: flex;
-  gap: 50px;
-  justify-content: center;
-  align-items: center;
-  flex-wrap: wrap;
-  text-decoration: none;
-  list-style: none;
-}
-
-.menu-part a {
-  text-decoration: none;
-  font-size: 25px;
-  line-height: 20px;
-  color: #344563;
-  font-weight: 500;
-  padding: 10px;
-  align-items: center;
-  font-style: bold;
-}
-
-.hamburgur {
-  padding: 10px;
-}
-
-.bar {
-  width: 30px;
-  height: 3px;
-  margin: 4px;
-  background: #6070ff;
-  display: flex;
-}
-
-section {
-  display: block;
-  margin: 0 auto;
-  max-width: 375px;
-}
-
-.background {
-  background: url(/images/header-shapes-mobile.png);
-  max-width: 375px;
-  background-repeat: no-repeat;
-  object-fit: cover;
-  height: 100%;
-  left: -34px;
-  margin-top: 2%;
-}
-
-.wrapper {
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  flex-direction: column;
-}
-
-.intro-text {
-  max-width: 375px;
-}
-
-.intro-text h1 {
-  font-weight: 700;
-  font-size: 48px;
-  line-height: 50px;
-  align-items: center;
-  letter-spacing: 0.37px;
-  color: #091e42;
-  margin-top: 15%;
-  padding: 0 7%;
-}
-
-.main-text {
-  grid-column: 3;
-  grid-row: 10;
-  padding: 7%;
-}
-
-.text-description {
-  font-weight: 400;
-  font-size: 20px;
-  line-height: 28px;
-  color: #344653;
-}
-
-.social {
-  align-self: flex-start;
-  margin-bottom: 5%;
-}
-
-.connection {
-  height: 24px;
-  font-size: 16px;
-  line-height: 24px;
-  color: #7f8cff;
-  font-weight: 900;
-  padding: 20px 0;
-}
-
-.media-icons {
-  display: flex;
-  align-content: center;
-  gap: 15px;
-  left: 6.4%;
-  bottom: 18.88%;
-  top: 77.81%;
-  width: 100%;
-  padding-top: 20px;
-  color: #505f79;
-}
-
-.social ul {
-  list-style: none;
-  margin-left: 5px;
-}
-
-.fa-brands {
-  color: #505f79;
-  font-size: 20px;
-  font-weight: 500;
-}
-
-.fa-solid {
-  color: #505f79;
-  font-size: 20px;
-  font-weight: 500;
-}
-
-.container {
-  margin: 0 auto;
-  max-width: 1000px;
-  height: 100%;
-  display: grid;
-  justify-content: center;
-  align-items: center;
-  text-align: center;
-  width: 100%;
-}
-
-.card {
-  box-sizing: border-box;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  gap: 12px;
-  width: 360px;
-  height: 620px;
-  background: #fff;
-  border: 1px solid #dfe1e6;
-  border-radius: 16px;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-  margin: 10px;
-}
```

Comment:
- Kindly, be sure your mobile version still sticks to the [Figma design](https://user-images.githubusercontent.com/52010789/206548470-985b609a-a380-452f-8a6c-26578dff7c6d.png) 👌👌 here your technology list overlaps on the description, please fix it 👀😱



| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/206549912-14fc5267-a43e-41de-becf-61c2d4b79e07.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/206549900-c6c0942f-1e63-4613-8320-01d79bfe99c6.png" width="200">|


😎️